opennms-doc/guide-admin/src/asciidoc/text/opentracing/setup.adoc

```diff
@@ -38,7 +38,8 @@ Enable tracing with Jaeger on _Minion_ by installing the `opennms-core-tracing-j
 echo 'opennms-core-tracing-jaeger' >> "$MINION_HOME/etc/featuresBoot.d/jaeger.boot"
 ----
 
-NOTE: Uninstalling the `opennms-core-tracing-jaeger` feature at runtime is not yet supported.
+NOTE: Either `sentinel-flows` or `sentinel-telemetry` need to be installed before installing `opennms-core-tracing-jaeger` feature.
```

Comment:
Do we need this note here about the Sentinel features in the "Enable tracing with Jaeger on Minion"?